### PR TITLE
Added my client

### PR DESCRIPTION
**Pull request by FlorentAnders**
To the Apps.md list.

### CODE REVIEW

**Comment by Gargron** [comment]
Concerning review:

> Warning! This app is a fork of Tusky that adds advertising and tracking code to what is offered by the original project as a completely free and privacy respecting Mastodon client. The interface of Panoramic is tweaked with some different colors and logos and some bits of the UI are moved around but it offers no fundamental feature advantages. Use Tusky instead to keep your Mastodon experience advertising free. 

Not sure if we should add this to the list of apps...

**Comment by ornithocoder** [comment]
I would recommend the developer to implement one or two features that _really distinguishes_ this new app from Tusky before adding it to the list.

One or two features that neither Tusky implements nor has on its roadmap - otherwise a pull request to that project would be encouraged/recommended - and that users would be willing to give up of an advertising free experience to have.

**Comment by FlorentAnders** [comment]
I'm okay with the idea of adding some distinguishing features. I currently do not have really much time for that but adding some was the plan of course. Shall I check back when I feel like it is ready?

**Comment by cwh1te** [comment]
My thinking is that, if ad-supported clients should even be listed here (and I'm personally leaning toward "no"), they should at least be in a separate category that clearly marks them as such.

File: Using-Mastodon/Apps.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Some people have started working on apps for the Mastodon API. Here is a list of
 |[dndkodon](https://play.google.com/store/apps/details?id=jp.coe.dndkodon)|N/A|Tsuyoshi Hyuga|
 |[Pawoo](https://play.google.com/store/apps/details?id=jp.pxv.pawoo)|N/A|[@pixiv@pawoo.net](https://pawoo.net/users/pixiv)|
 |[Stegodon](https://play.google.com/store/apps/details?id=com.gilawhost.stegolophodon)|N/A|[@GilaW@mastodon.cloud](https://mastodon.cloud/@Gilaw)|
+|[Panoramic](https://play.google.com/store/apps/details?id=com.fa.imaged)|<https://github.com/FlorentAnders/Panoramicy>|[@florentanders@social.tchncs.de](https://social.tchncs.de/web/accounts/12666)|
```
Comment (jjatria) [inline Using-Mastodon/Apps.md]: Regardless of the rest of the discussion in this PR, the link should point to <https://github.com/FlorentAnders/Panoramic>. The link currently has an extra `y`, and is broken.

**Pull request closed without merging**